variables.tf

```diff
@@ -32,8 +32,11 @@ variable "set_roles" {
 }
 
 variable "per_folder_admins" {
-  type        = map(string)
-  description = "IAM-style members per folder who will get extended permissions."
+  type = map(object({
+    member = string
```

Comment:
Since we support multiple members with comma delim, it would be a good ideas to make this a `members` list(string) so we can accurately type type this.
https://github.com/terraform-google-modules/terraform-google-folders/blob/40c8bae3b96707b7054e13a05bae87cc2dd58726/main.tf#L49

Reply:
done